src/BenchmarkDotNet/Validators/DotNetSdkVersionValidator.cs

```diff
@@ -0,0 +1,326 @@
+﻿using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Running;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace BenchmarkDotNet.Validators
+{
+    public static class DotNetSdkVersionValidator
+    {
+        private static readonly Lazy<List<string>> cachedFrameworkSdks =
+        new Lazy<List<string>>(() => GetInstalledFrameworkSdks().ToList(), true);
```

Comment:
Just return the expected type from the function.

```suggestion
        private static readonly Lazy<List<string>> cachedFrameworkSdks = new Lazy<List<string>>(GetInstalledFrameworkSdks, true);
```